app/lib/surface/Mark.coffee

```diff
@@ -219,7 +219,7 @@ module.exports = class Mark extends CocoClass
     shapeName = if @lank.thang.shape in ['ellipsoid', 'disc'] then 'ellipse' else 'rect'
     key = "#{shapeName}-debug-#{@lank.thang.collisionCategory}"
     DEBUG_SIZE = 10
-    unless key in @layer.spriteSheet.getAnimations()
+    unless key in @layer.spriteSheet.animations
```

Comment:
Are all these changes from `getAnimations()` to `animations` because this function is costly?

Reply:
The `get` style are deprecated and throw warnings in CreateJS 1.0, recommended to just use accessors directly.